instance/tests/test_tasks.py

```diff
@@ -59,6 +61,11 @@ def test_watch_pr_new(self, mock_get_username_list, mock_get_pr_list_from_userna
         """
         New PR created on the watched repo
         """
+        ansible_extra_settings = textwrap.dedent("""\
+            WATCH: true
+            edx_ansible_source_repo: https://github.com/open-craft/configuration
+            configuration_version: named-release/elder
+        """)
```

Comment:
Why do you need to add these variables in this test? Do we still have a test that still uses all the default values, if we set `edx_ansible_source_repo` and `configuration_version` in this one?